astropy/coordinates/transformations/graph.py

```diff
@@ -483,6 +483,8 @@ def to_dot_graph(
         """
         # We store nodes as `dict` keys because differently from `list` uniqueness is
         # guaranteed and differently from `set` insertion order is preserved.
+        if addnodes is None:
+            addnodes = []
```

Comment:
`addnodes` is only used in a single line, so it's better to have the `if addnodes is None` check just before that line.